docs/xgcpolicy.md

```diff
@@ -42,32 +42,31 @@ Controls the behavior of the garbage collector by specifying different garbage c
 | [`nogc`](#nogc)                                                              |         |
 
-
-Specify the garbage collection policy that you want the OpenJ9 VM to use:
+For a detailed description of the policies, when to use them, and how they work, see [Garbage Collection policies](gc.md). The following GC policies are available:
 
 ### `gencon`
 
         -Xgcpolicy:gencon
 
-: The generational concurrent policy (default) uses a concurrent mark phase combined with generational garbage collection to help minimize the time that is spent in any garbage collection pause. This policy is particularly useful for applications with many short-lived objects, such as transactional applications. Pause times can be significantly shorter than with the `optthruput` policy, while still producing good throughput. Heap fragmentation is also reduced.
+: The generational concurrent policy (default) requires a heap that is divided into two main areas (*nursery* and *tenure*) to manage two generation groups (*new* and *older*). The policy uses a global GC cycle of concurrent *mark-sweep* operations, optionally followed by *compact* operations. The policy also uses a partial GC cycle to run *scavenge* operations on the *nursery* area. The partial cycle helps reduce the frequency and duration of the global GC cycle. Note that *scavenge* is a *stop-the-world* operation, unless `-Xgcpolicy:gencon` is specified with the [`-Xgc:concurrentScavenge`](xgc.md#concurrentscavenge) option (supported only on IBM Z&reg; systems).
 
-: To learn more about this policy and how it works, see [Garbage collection: 'gencon' policy](gc.md#gencon-policy-default).
+: To learn more about this policy, when to use it, and how it works, see [Garbage collection: `gencon` policy](gc.md#gencon-policy-default).
 
 
 ### `balanced`
 
         -Xgcpolicy:balanced
 
-: The balanced policy policy uses mark, sweep, compact and generational style garbage collection. The concurrent mark phase is disabled; concurrent garbage collection technology is used, but not in the way that concurrent mark is implemented for other policies. The `balanced` policy uses a region-based layout for the Java&trade; heap. These regions are individually managed to reduce the maximum pause time on large heaps and increase the efficiency of garbage collection. The policy tries to avoid global collections by matching object allocation and survival rates.
+: The Balanced policy requires a multi-region heap to manage multiple generations of objects. The policy uses a global GC cycle that involves an incremental concurrent *mark* operation (global mark phase), followed by *stop-the-world* (STW) *sweep* operation. The policy also uses a partial GC cycle to run *copy forward* or *mark-compact* operations. Regions are individually managed to reduce the maximum pause time on large heaps and increase the efficiency of garbage collection. The policy tries to avoid global collections by matching object allocation and survival rates.
```

Comment:
You may add a detail that the global sweep followed by GMP is actually performed within the same increment of the first PGC after GMP is completed (not as as a part of last GMP cycle increment, as it's done in Gencon)

Reply:
I'm not sure I understand you completely @amicic, so please correct me if this sentence is wrong:

"With the `balanced` policy, the global and partial GC cycles interleave. That is, the global STW *sweep* operation runs within the same GC increment as the partial GC cycle."